xdsl/dialects/stim/stim_printer_parser.py

```diff
@@ -19,6 +23,32 @@ def in_braces(self):
         finally:
             self.print_string("}")
 
+    @contextmanager
+    def in_parens(self):
+        self.print_string("(")
+        try:
+            yield
+        finally:
+            self.print_string(")")
```

Comment:
This was never required, I'm not sure why we have this pattern
```suggestion
        yield
        self.print_string(")")
```